src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -361,6 +361,7 @@
   "DownloadClientRootFolderHealthCheckMessage": "Download client {0} places downloads in the root folder {1}. You should not download to a root folder.",
   "DownloadClientSettings": "Download Client Settings",
   "DownloadClientSortingHealthCheckMessage": "Download client {0} has {1} sorting enabled for Sonarr's category. You should disable sorting in your download client to avoid import issues.",
+  "DownloadClientHistoryRetentionHealthCheckMessage": "Download client {0} does not have History Retention enabled. You should enable this in your download client to ensure completed downloads can be imported.",
```

Comment:
I think this should be more generic to cover: https://github.com/Sonarr/Sonarr/issues/5343

Perhaps `Download client {0} is set to remove completed downloads. This can result in downloads being removed from your client before Sonarr can import them.`

Reply:
reworked this